pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -149,6 +149,7 @@ func poolMinioServerContainer(t *miniov2.Tenant, skipEnvVars map[string][]byte,
 
 	containerPorts := []corev1.ContainerPort{
 		{
+			Name:          miniov2.MinIOPortName,
```

Comment:
If we name the MinIO port, then it makes sense to also name the console port (a few lines down).